### ISSUE TITLE
Open "Edit Styles" dialog fails with 6.4.0 (after upgrade from 6.3.0)

### ISSUE THREAD

**Issue by bbak** [stevencohn/OneMore#1420]
# Problem to Solve

Edit Styles dialog doesn't Open

# To Reproduce

Try to open "Edit Styles" Dialog.

# Application Log

```
13|12:24:34.700| Running command EditStylesCommand
13|12:24:34.905| ..cannot translate FileMenu, name not found
14|12:24:42.891| ..Running command EditStylesCommand
14|12:24:43.003| ..cannot translate FileMenu, name not found
15|12:24:49.697| ..Running command LoadStylesCommand
05|12:24:57.263| ..hashtag service scanned 2266 pages, updating 0, in 109101ms
15|12:24:58.134| ..loaded theme Standard
07|12:25:01.623| Running command EditStylesCommand
07|12:25:01.724| ..cannot translate FileMenu, name not found
14|12:25:26.979| ..Running command LoadStylesCommand
04|12:25:27.014| Fehler beim Ausführen des Befehls EditStylesCommand
04|12:25:27.014| System.ArgumentNullException
Message: Der Wert darf nicht NULL sein.
Parametername: value
Source: System.Windows.Forms
StackTrace:    bei System.Windows.Forms.ComboBox.ObjectCollection.IndexOf(Object value)
   bei River.OneMoreAddIn.Commands.StyleDialog.Initialize()
   bei River.OneMoreAddIn.Commands.StyleDialog..ctor(Theme theme, Color pageColor, Boolean darkMode)
   bei River.OneMoreAddIn.Commands.EditStylesCommand.<Execute>d__1.MoveNext()
--- Ende der Stapelüberwachung vom vorhergehenden Ort, an dem die Ausnahme ausgelöst wurde ---
   bei System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   bei River.OneMoreAddIn.Commands.EditStylesCommand.<Execute>d__1.MoveNext()
--- Ende der Stapelüberwachung vom vorhergehenden Ort, an dem die Ausnahme ausgelöst wurde ---
   bei System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   bei System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   bei System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   bei River.OneMoreAddIn.CommandFactory.<Run>d__6.MoveNext()
TargetSite: [System.Windows.Forms] System.Windows.Forms.ComboBox+ObjectCollection::IndexOf()
```

# Environment (if applicable)

 - OneNote Version: Microsoft® OneNote® für Microsoft 365 MSO (Version 2404 Build 16.0.17531.20152) 64 Bit 
 - OneMore Version 6.4.0; upgraded by removing 6.3.0 first (which deleted my modified styles), since inplace upgrade failed several times in the past.

**Comment by ykurji** [comment]
Facing the exact same issue

**Comment by bogaladyte** [comment]
So until next release the only way to fix this is to switch to 6.3?

**Comment by stevencohn** [comment]
Yes. Hoping to get a new version out this week with some bug fixes including this one.
